ishlib/gpu/searcher_device.mojo

```diff
@@ -157,7 +157,9 @@ struct SearcherDevice[func_type: AnyTrivialRegType, //, func: func_type]:
         max_devices: Int,
     ) raises -> List[Self]:
         var ret = List[Self]()
-        for i in range(0, min(DeviceContext.number_of_devices(), max_devices)):
+        for i in range(
```

Comment:
Hardcoded to max of 1 for now. Warning will happen in SearcherSettings if you set it above 1. This is because there isn't a good way to filter out non-compatible GPUs.